lib/process-services/src/lib/form/services/process-content.service.ts

```diff
@@ -231,11 +231,12 @@ export class ProcessContentService {
     handleError(error: any): Observable<any> {
         let errMsg = ProcessContentService.UNKNOWN_ERROR_MESSAGE;
         if (error) {
-            errMsg = (error.message) ? error.message :
-                error.status ? `${error.status} - ${error.statusText}` : ProcessContentService.GENERIC_ERROR_MESSAGE;
+            errMsg = error.message
```

Comment:
We may want to fix this sonarcloud issue here: 
<img width="963" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/cc84150b-b782-47c1-93c5-72d21b01c5fe">

Reply:
it's better fixing this separately, as Sonar will keep finding more stuff, and this PR risks to turn into the overall fix